app/styles/_forms.scss

```diff
@@ -73,6 +73,15 @@ form {
     height: 30px;
   }
 
+  input[type="submit"]{
+      background-color: grey;
+      border: grey;
+      color: #ffffff;
+      &:focus, &:hover{
+      background-color: black;
+    }
+  }
```

Comment:
This is invalid SCSS (you have one open `{` and two close `}`s)

Reply:
My apologies for missing out the indentation which makes it looks invalid. &:focus, &:hover are actually wrapped by the "input[type="submit"]", so the second closing } is for the input tag